lib/krikri/registry.rb

```diff
@@ -0,0 +1,45 @@
+module Krikri
+
+  ##
+  # Provide a registry of defined items that can be looked up by a token
```

Comment:
Re: your first point, I agree with @no-reply's comment in Slack: 

```
< tom> ah, seeing the database issue, i'm now more in favor of #to_sym
``` 

Regarding your second point (UI issues of harvester types), it seems like that ideally we'd best be dealing with that through defining locale files for `I18n`.